packages/odyssey-react/src/components/Tabs/Tabs.module.scss

```
@@ -12,43 +12,44 @@
 
 .tablist {
   position: relative;
-  border-block-end: 1px solid $border-color-display;
+  border-block-end-width: var(--BorderBlockEndWidthTabList);
+  border-block-end-style: var(--BorderBlockEndStyleTabList);
+  border-block-end-color: var(--ColorBorderBlockEndTabList);
 }
 
 .tab {
   display: inline-block;
   position: relative;
-  margin-block: 0;
-  margin-inline: 0;
-  padding-block: $spacing-s;
-  padding-inline: $spacing-s;
+  margin-block: var(--SpaceMarginTab);
+  margin-inline: var(--SpaceMarginTab);
+  padding-block: var(--SpacePaddingTab);
+  padding-inline: var(--SpacePaddingTab);
```

@francoislehoux-okta we'll want to break these up into separate component variables rather than applying the same variable to multiple style declarations. So something like this:

```suggestion
  margin-block: var(--SpaceMarginBlockTab);
  margin-inline: var(--SpaceMarginInlineTab);
  padding-block: var(--SpacePaddingBlockTab);
  padding-inline: var(--SpacePaddingInlineTab);
```